starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -1,9 +1,12 @@
 package com.starter.fullstack.dao;
 
 import com.starter.fullstack.api.Inventory;
+
 import java.util.List;
 import java.util.Optional;
 import javax.annotation.PostConstruct;
+import javax.validation.constraints.Null;
+
```

Comment:
although this is very small, you always will want to make sure the imports are together and not separated. If you have them separated then your build will fail because of this

Reply:
Resolved by keeping all imports together